src/main/webapp/onelive/functions.js

```diff
@@ -1,3 +1,40 @@
-// Write custom js functions for the onelive page here
-
+//function to load upcoming events
+function loadUpcomingEvents() {
+    $.ajax({
+        type: 'GET',
+        url: 'https://sef-dataholder.herokuapp.com/calendar/onelive?maxResults=3',
+        dataType: 'json',
+        success: function (payload) {
+            if (payload.items === undefined || payload.items.length == 0) {
+                $("#section-events").hide();
+            } else {
+                payload.items.forEach((item) =>{
+                    /*todo
+                    * to get img link
+                    * let fileId = item.attachments[0].fileId;
+                    * item.imgLink = "https://drive.google.com/uc?export=view&id=" + fileId;
+                    * */
 
+                    //get fb link and remove it from description (split the part of the description where the fb link is added)
+                    let description = item.description;
+                    item.facebookLink = description.substring(description.lastIndexOf('<a href') + 9, description.lastIndexOf('" id='));
```

Comment:
`description.lastIndexOf('<a href') + 9, description.lastIndexOf('" id=')`

What does this code snippet do?  (9 ?)

Reply:
lastIndexOf function returns the location of "<" sign of "<a href =...." . But the link starts after 9 characters. So i added 9 to get the starting point of link

Reply:
oh got it